Testscripts/Linux/utils.sh

```diff
@@ -2626,6 +2626,80 @@ function install_ntttcp () {
 	fi
 }
 
+# Install apache and required packages
+function install_apache () {
+	LogMsg "Detected $DISTRO_NAME $DISTRO_VERSION; installing required packages of apache"
+	update_repos
+	case "$DISTRO_NAME" in
+		oracle|rhel|centos)
+			install_epel
+			yum clean dbcache
+			yum -y --nogpgcheck install sysstat zip httpd httpd-tools dstat
+			;;
+
+		ubuntu|debian)
+			dpkg_configure
+			install_package "libaio1 sysstat zip apache2 apache2-utils dstat"
+			;;
+
+		sles|sle_hpc)
```

Comment:
why is it 'sle_hpc'? I thought we need 'suse'

Reply:
Fixed.